kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/settings/DeviceCertsTabUi.java

```diff
@@ -231,4 +231,9 @@ private boolean isDeviceCertValid() {
             return true;
         }
     }
+
+    @Override
```

Comment:
Please change the header to: " * Copyright (c) 2011, 2020 Eurotech and/or its affiliates"